.github/workflows/vast.yaml

```diff
@@ -506,14 +506,14 @@ jobs:
           # Zero the cache statistics (but not the configuration options).
           ccache --zero-stats
           ccache --show-config
-          ./configure \
-            --prefix="${PWD}/opt/vast" \
-            --build-dir="$BUILD_DIR" \
-            --package-name="$PACKAGE_NAME" \
-            --with-pcap-plugin \
-            --with-lsvast \
-            --with-dscat \
-            --with-bundled-caf \
+          cmake -B "$BUILD_DIR" \
+            -DCMAKE_INSTALL_PREFIX="${PWD}/opt/vast" \
+            -DCPACK_PACKAGE_FILE_NAME="$PACKAGE_NAME" \
+            -DCPACK_GENERATOR:STRING=TGZ \
+            -DVAST_PLUGINS:STRING="${PWD}/plugins/pcap" \
```

Comment:
Why do we build with PCAP on macOS but without it on Debian?

Reply:
No idea; probably a copy/pasting mistake but not something I want to touch in this PR.